src/main/webapp/components/AcceptModal.test.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+import AcceptModal from './AcceptModal';
+
+describe('AcceptModal', () => {
+    let wasAccepted = false;
```

Comment:
I would use jest.fn function
[example in the code](https://github.com/splunk/addonfactory-ucc-base-ui/blob/b8d44af77195a5a3f02f73c9ede59a5ab124377d/src/main/webapp/components/MenuInput.test.tsx#L109C1-L112C12)

p.s. it is better to assign variables in `beforeEach` or `it` to avoid side effects

Reply:
yea, it was more like declaration but yea functions suits here much better.